cdbot/cogs/cyber.py

```diff
@@ -359,33 +359,17 @@ def __init__(self, bot: Bot):
         """
         if ELITECOUNT_ENABLED:
             preferences = {
-                'London': {
-                    'Younger': Roles.Elite.London.YOUNGER,
-                    'Older': Roles.Elite.London.OLDER
+                '2019': {
+                    'Cyberists': Roles.Elite.VET2019.CYBERIST,
+                    'Forensicators': Roles.Elite.VET2019.FORENSICATOR
                 },
-                'Birmingham': {
-                    'Younger': Roles.Elite.Birmingham.YOUNGER,
-                    'Older': Roles.Elite.Birmingham.OLDER
-                },
-                'Lancaster': {
-                    'Younger': Roles.Elite.Lancaster.YOUNGER,
-                    'Older': Roles.Elite.Lancaster.OLDER
-                }
             }
 
-            elite = len(ctx.guild.get_role(Roles.Elite.MAIN).members)
-            exchange = len(ctx.guild.get_role(Roles.Exchange.SHORTLIST).members)
-            confirmed = len(ctx.guild.get_role(Roles.Exchange.CONFIRMED).members)
-
-            year = datetime.datetime.utcnow().year
-
             description = textwrap.dedent(f"""
-            **Total qualified**: {elite}
-
             **Camp Statistics**
             """)
 
-            embed = Embed(title=f"CyberStart Elite {year}",
+            embed = Embed(title=f"CyberStart Elite {datetime.datetime.utcnow().year}",
```

Comment:
Not sure about the idea of having it show current year here, although not material to this commit